src/main/java/ch/njol/skript/expressions/ExprEntities.java

```diff
@@ -56,19 +57,20 @@
  * @author Peter Güttinger
  */
 @Name("Entities")
-@Description("All entities in all worlds, in a specific world or in a radius around a certain location, " +
+@Description("All entities in all worlds, in a specific world, in a chunk or in a radius around a certain location, " +
 		"e.g. 'all players', 'all creepers in the player's world', or 'players in radius 100 of the player'.")
 @Examples({"kill all creepers in the player's world",
-		"send \"Psst!\" to all players witin 100 meters of the player",
+		"send \"Psst!\" to all players within 100 meters of the player",
 		"give a diamond to all ops",
-		"heal all tamed wolves in radius 2000 around {town center}"})
-@Since("1.2.1")
+		"heal all tamed wolves in radius 2000 around {town center}",
+		"delete all monsters in chunk at player"})
+@Since("1.2.1, INSERT VERSION (chunks)")
 public class ExprEntities extends SimpleExpression<Entity> {
 
 	static {
 		Skript.registerExpression(ExprEntities.class, Entity.class, ExpressionType.PATTERN_MATCHES_EVERYTHING,
-				"[(all [[of] the]|the)] %*entitydatas% [(in|of) [world[s]] %-worlds%]",
-				"[(all [[of] the]|the)] entities of type[s] %entitydatas% [(in|of) [world[s]] %-worlds%]",
+				"[(all [[of] the]|the)] %*entitydatas% [(in|of) (0¦%-worlds%|1¦%-chunks%)]",
```

Comment:
Removing a word could potentially cause syntax conflicts for no reason. Also, you only need a single mark here, it's 0 by default after all.

Reply:
I removed world because blue told me in the past that the world expression already used it. 
So it being there would be used like `all entities in world world "world"` which seemed pointless. But I can put it back if you truly feel its necessary to keep it. I do see your point although I really dont think anyones typing it that way.